src/main/webapp/.babelrc

```diff
@@ -9,11 +9,6 @@
         "style": true
       }
     ],
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-transform-react-jsx-source",
```

Comment:
These are include by babel-preset-react when development mode is set. (https://babeljs.io/docs/en/babel-preset-react#development) Not sure how I can set this though, since I don't want it hardcoded.

Reply:
Fixed in 96ed3c7